azure-toolkit-libs/azure-toolkit-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/lib/springcloud/service/SpringCloudClusterService.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.springcloud.service;
+
+import com.microsoft.azure.PagedList;
+import com.microsoft.azure.management.appplatform.v2020_07_01.ServiceResource;
+import com.microsoft.azure.management.appplatform.v2020_07_01.implementation.AppPlatformManager;
+import com.microsoft.azure.management.appplatform.v2020_07_01.implementation.AppResourceInner;
+import com.microsoft.azure.management.appplatform.v2020_07_01.implementation.ServiceResourceInner;
+import com.microsoft.azure.toolkit.lib.springcloud.model.SpringCloudAppEntity;
+import com.microsoft.azure.toolkit.lib.springcloud.model.SpringCloudClusterEntity;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class SpringCloudClusterService {
+
+    private final AppPlatformManager client;
+
+    public SpringCloudClusterService(AppPlatformManager client) {
+        this.client = client;
+    }
+
+    public void remove(final SpringCloudClusterEntity service) {
+        this.client.services().deleteAsync(
+            service.getResourceGroup(),
+            service.getName()).await();
+    }
+
+    public SpringCloudClusterEntity reload(final SpringCloudClusterEntity service) {
```

Comment:
where's the getSpringCloudCluster method? I need to create SpringCloudClusterEntity for a get purpose?